dir.traversal.targets

```diff
@@ -122,7 +122,7 @@
       </Project>
 
       <!-- for projects not defining TestTFM in .builds files, we assume the TestTFM is the default which is netcoreapp1.0 -->
-      <ProjectsToTest Include="@(Project)" Condition="'%(Project.Extension)'=='.csproj' And '$(FilterToTestTFM)' == 'netcoreapp1.0' And '%(Project.TestTFMs)' == ''" />
+      <ProjectsToTest Include="@(Project)" Condition="'%(Project.Extension)'=='.csproj' And '$(FilterToTestTFM)' == 'netcoreapp1.0' And '%(Project.TestTFMs)' == '' And '%(Project.SkipTestRun)'!='true'" />
```

Comment:
Removed

---
In reply to: [72678855](https://github.com/dotnet/corefx/pull/10297#discussion_r72678855) [](ancestors = 72678855)